src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -181,6 +181,7 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
+      const char* return_info = NULL;
```

Comment:
@OHerasym, don't be afraid to use string be value. Even non-const. By the way, assigning something to the **const** pointer looks very strange.